Makefile

```
@@ -1,11 +1,20 @@
-.PHONY: testshort
-testshort:
+help: ## Print usage
+	@sed -r '/^(\w+):[^#]*##/!d;s/^([^:]+):[^#]*##\s*(.*)/\x1b[36m\1\t:\x1b[m \2/g' ${MAKEFILE_LIST}
```

This pattern doesn't allow hyphens in the target name.

Suggest adding that

```suggestion
	@sed -r '/^\S+:[^#]*##/!d;s/^(\S+):[^#]*##\s*(.*)/\x1b[36m\1\t:\x1b[m \2/g' ${MAKEFILE_LIST}
```

---

added